core/txpool/celo_validation.go

```diff
@@ -86,6 +86,12 @@ func CeloValidateTransaction(tx *types.Transaction, head *types.Header,
 			if err != nil {
 				return err
 			}
+			// When a currency is worth more than celo it can happen the the celo denominated min
+			// tip results in a 0 min tip for the currency. We enforce that there must always be a
+			// non zero tip.
+			if minTip.Sign() == 0 {
+				minTip = big.NewInt(1)
+			}
```

Comment:
Since this is only in the txpool, it is not a hard fork. But we still reject some txs that we allowed earlier. Are we sure we don't break some applications if we deploy this on mainnet? If not, do we need to warn users well ahead of deploying this?